Updated flake physics, suites and MOM6-interface

YihuaWu-NOAA opened the pull request:
## Description

(Instructions: this, and all subsequent sections of text should be removed and filled in as appropriate.)
Provide a detailed description of what this PR does.
What bug does it fix, or what feature does it add?
Is a change of answers expected from this PR?
Are any library updates included in this PR (modulefiles etc.)?

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues should always be created before starting work on a PR branch!)
- fixes #<issue_number>
- fixes noaa-emc/fv3atm/issues/<issue_number>

## Testing

How were these changes tested?
What compilers / HPCs was it tested with?
Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)
Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

## Dependencies

If testing this branch requires non-default branches in other repositories, list them.
Those branches should have matching names (ideally)

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs
- waiting on ncar/ccpp-physics/pull/588
- waiting on noaa-emc/fv3atm/pull/256

junwang-noaa commented:
@YihuaWu-NOAA Please fill in information in the description section. If you created an issue, please list the issue number, if not, please create a new issue about your code changes. In Testing section, please list what test you have done, and if RT passes on the platforms you have access to. In the dependency section, please list subcomponent PRs (e.g. f3atm, ccpp etc) associated with this PR.

YihuaWu-NOAA commented:
Hi, Jun,

Thanks!  I updated flake driver, and modified sfc_nst.f & sfc_nst.meta as
well as module_nst_water_prop.f90 to seperate lakes from the ocean.
I am rerunning the regression tests.  I will fill in this part after they
are done.




*Yihua----------------------------------------------------Yihua Wu, Ph D*I.M.
Systems Group
NOAA/NWS/NCEP/EMC
5830 University Research Court, Room: 2031
College Park, Maryland 20740, USA
Phone: (301)-683-3691
Fax:       (301)-683-3703


On Tue, Mar 9, 2021 at 8:12 AM Jun Wang <notifications@github.com> wrote:

> @YihuaWu-NOAA <https://github.com/YihuaWu-NOAA> Please fill in
> information in the description section. If you created an issue, please
> list the issue number, if not, please create a new issue about your code
> changes. In Testing section, please list what test you have done, and if RT
> passes on the platforms you have access to. In the dependency section,
> please list subcomponent PRs (e.g. f3atm, ccpp etc) associated with this PR.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/454#issuecomment-793879828>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANJBYQELT7BU6TT4QSEVXTDTCYGCJANCNFSM4YZ2RNUA>
> .
>

BrianCurtis-NOAA commented:
new one made with connected repo, closing old one (new one #506)
